src/python/grapl_analyzerlib/grapl_analyzerlib/grapl_client.py

```diff
@@ -59,8 +59,8 @@ def txn_context(
     def alter(
         self,
         operation: Operation,
-        timeout: Optional[SecsDuration] = None,
-        metadata: Optional[DgraphMetadata] = None,
+        timeout: SecsDuration | None = None,
+        metadata: DgraphMetadata | None = None,
         credentials: Optional[CallCredentials] = None,
```

Comment:
This CallCredentials being rewritten causes sphinx to break because CallCredentials doesn't support the pipe operator. 👎